src/main/resources/swagger.api/schemas/entityTypeColumn.json

```diff
@@ -24,6 +24,10 @@
         "sourceAlias": {
           "description": "Source alias for this column",
           "type": "string"
+        },
+        "hidden": {
+          "description": "Indicates if the field is hidden in UI",
+          "type": "boolean"
```

Comment:
Might as well make the default explicit (and with any kind of luck, that will cause the resulting data type to be `boolean` instead of `Boolean`)